packages/core/admin/admin/src/content-manager/pages/EditSettingsView/index.js

```diff
@@ -322,6 +329,7 @@ const EditSettingsView = ({ mainLayout, components, isContentTypeView, slug, upd
                       type: 'ON_ADD_FIELD',
                       name: field,
                       fieldSizes,
+                      getCustomField: customFieldsRegistry.get,
```

Comment:
I don't think you should do this, if you need all the components, either you need to pass them all or refactor to avoid it. Aren't actions serialised? you're also not using it in the action :)

Finally, why are you passing `fieldSizes`? If it's already in redux, you can access it there no?

Reply:
I believe I need to pass `fieldSizes` because it's in another "slice", I don't have access to it as it's outside of the scope of this state.

Regarding getCustomField, I forgot to remove it after changing my implementation, thanks for the heads up. And I didn't know about the Redux serialisation rules, so also thanks :)